luigi/worker.py

```diff
@@ -238,8 +238,13 @@ class DequeQueue(collections.deque):
     deque wrapper implementing the Queue interface.
     """
 
-    put = collections.deque.append
-    get = collections.deque.pop
+    def put(self, obj, block=None, timeout=None):
+        del block, timeout
```

Comment:
Why delete these variables? It should be sufficient to ignore them. Does this line do anything?